assets/js/components/Collection/Form.jsx

```diff
@@ -83,64 +57,53 @@ const CollectionForm = ({ collection }) => {
     history.push("/collection/list");
   };
 
-  const handleInputChange = e => {
-    setFormValues({
-      ...formValues,
-      [e.target.name]: e.target.value
-    });
-  };
-
-  const handleIsFeaturedChange = e => {
-    const prevVal = formValues.featured;
-    setFormValues({
-      ...formValues,
-      featured: !prevVal
-    });
-  };
-
-  const handleSubmit = e => {
-    e.preventDefault();
-
-    // Convert keywords to an array as defined in GraphQL
-    let values = { ...formValues };
-    values.keywords = formValues.keywords
-      .split(",")
-      .map(keyword => keyword.trim());
-
+  const onSubmit = data => {
     if (!collection) {
       createCollection({
-        variables: { ...values }
+        variables: { ...data }
       });
     } else {
       updateCollection({
-        variables: { ...values, collectionId: collection.id }
+        variables: { ...data, collectionId: collection.id }
       });
     }
   };
 
   return (
     <div>
-      <form onSubmit={handleSubmit} data-testid="collection-form">
+      <form onSubmit={handleSubmit(onSubmit)} data-testid="collection-form">
         <div className="columns is-centered">
           <div className="column is-half">
-            <UIFormInput
-              placeholder="Add collection Name"
-              type="text"
-              name="collectionName"
-              id="collectionName"
-              label="Collection Name"
-              value={formValues.collectionName || ""}
-              onChange={handleInputChange}
-              data-testid="collection-name"
-            />
+            <div className="field">
+              <label htmlFor="collectionName" className="label">
```

Comment:
This might have been my bad initially setting up, but the `aria` `htmlFor`attribute targets an `id`, not a `name` attribute.  Can you update the `<label>` elements everywhere in this form to look like:

```
<label htmlFor="[id of element]"...
```